deltaspike/modules/data/impl/src/main/java/org/apache/deltaspike/data/impl/handler/QueryHandler.java

```diff
@@ -154,10 +154,6 @@ public Object process(Object proxy, Method method, Object[] args) throws Throwab
         catch (Exception e)
         {
             log.log(Level.FINEST, "Query execution error", e);
-            if (queryContext != null)

```

Comment:
Everything else in this PR looks fine except this change.  There is a chance that queryContext may be null here, its just not reported due to runtime exceptions.